.github/workflows/release-pipeline.yaml

```diff
@@ -68,6 +68,7 @@ jobs:
     name: Deploy Production
     uses: ./.github/workflows/deploy.yaml
     needs: [ integration-tests, set-version ]
+    if: always()
```

Comment:
I think we can do this `if: ${{jobs.integration-tests.result = "success" || jobs.integration-tests.result = "failure"}}` this means it will not run if it's been skipped or cancelled. We could do if not skipped and not cancelled in the logic also, not sure which is more expressive.